includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -3129,6 +3130,136 @@ static function ( $attribute_name ) {
 				}
 				return $title;
 
+			case AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_CHILD_TAG:
+				return sprintf(
+					/* translators: %1$s is the child tag, %2$s is node name */
+					__( 'Tag <code>%1$s</code> is disallowed as child of tag <code>%2$s</code>', 'amp' ),
+					esc_html( $validation_error['child_tag'] ),
+					esc_html( $validation_error['node_name'] )
```

Comment:
The markup doesn't need to be in the translation string, so:

```suggestion
					esc_html__( 'Tag %1$s is disallowed as child of tag %2$s', 'amp' ),
					'<code>' . esc_html( $validation_error['child_tag'] ) . '</code>',
					'<code>' . esc_html( $validation_error['node_name'] ) . '</code>'
```

Note also that this allows `esc_html__()` to be used.

Reply:
Same goes for instances below.